Articles of Association.md

```diff
@@ -0,0 +1,57 @@
+4. General Assembly
+    1. The General Assembly is the highest authority of the association.
+    2. The Annual General Assembly is held electronicly once a year before the end of ______ and is convened with at least three weeks' notice to members.
+    3. The public is eligible to attend the General Assembly.
+    4. All members who are actively contributing towards the purpose stipulated in section 2 are eligible to vote.
+    5. The agenda for the annual General Assembly must contain at least the following items:
+        1. Election of the Speaker
+        2. Election of secretary
+        3. Determination of present members
+        4. Financial reporting
+        5. Consideration of proposals received
+        6. Determination of membership fee
+        7. Approval of the budget
+        8. Election of chairman
+        9. Election of deputy chairman
+        10. Election of 3-7 board members and 2 substitutes
+        11. Election of auditor (s) and deputy auditor 
+        12. Any other business
+    6. Proposals that are to be considered at the General Assembly must be received by the Board of Directors no later than 8 days before the General Assembly. Proposals for amendments to the articles of association that are to be considered must be received by the Board of Directors no later than three weeks before the General Assembly.
+    7. The General Assembly is presided by the Speaker who cannot be a member of the Board of Directors.
+    8. The General Assembly makes its resolutions by 1/2 of votes and by electronic voting.
+    9. Voting can be done anonymously if 1/3 of the Board of Directors is in favour.
+5. Extraordinary General Assembly
+    1. Extraordinary General Assembly may be held when the Board of Directors deems it necessary or when at least 1/3 of the members submit a written reasoned request to the chairman and deputy chairman. In such cases, the General Assembly must be held no later than four weeks after the request has come to the knowledge of the chairman or deputy chairman.
+6. The daily management of the association
+    1. The daily management of the association consists of the board, which in addition to the chairman consists of the deputy chairman, the treasurer and two to six additional members. The board of directors is elected by the General Assembly.
+    2. The board of directors leads the association in accordance with these articles of association and the resolutions of the General Assembly.
+    3. No later than 14 days after re-electing the Board of Directors, a constituent meeting is held, where the Board of Directors elects the treasurer from among its members. All positions of trust are valid until the next General Assembly.
```

Comment:
"All positions of trust are valid until the next General Assembly." It might be wise to allow for a re-election of the board of directors at an Extraordinary General Assembly?